dev-haskell/reflex/reflex-0.9.3.3.ebuild

```diff
@@ -21,29 +21,29 @@ IUSE="debug plugin +template-haskell"
 CABAL_CHDEPS=(
 	'hlint (< 2.1 || >= 2.2.2) && < 3.5' 'hlint >= 2.2.2'
 	'hlint >= 3.5 && < 3.6' 'hlint >= 3.5'
-	'constraints >= 0.10 && <0.14' 'constraints >= 0.10'
-	'primitive >= 0.5 && < 0.8' 'primitive >= 0.5'
-	'lens >= 4.7 && < 5.3' 'lens >= 4.7'
+	'constraints >= 0.10 && <0.15' 'constraints >= 0.10'
+	'primitive >= 0.5 && < 0.10' 'primitive >= 0.5'
+	'lens >= 4.7 && < 5.4' 'lens >= 4.7'
 )
```

Comment:
General comment: you don't need to keep these `CABAL_CHDEPS` changes if they're unnecessary based on the latest version in the repo.

Particularly in this case, notice that you're unbounding the dependency range in `cabal_chdeps` but the same dependencies are upper-bounded in the RDEPENDs.

This particular case does no harm, so don't change it, it's fine. Just food for thought in future bumps.